packages/loot-design/src/components/modals/NordigenExternalMsg.js

```diff
@@ -9,25 +9,30 @@ import Autocomplete from '../Autocomplete';
 import { View, Modal, Button, P } from '../common';
 import { FormField, FormLabel } from '../forms';
 
-function useAvailableBanks() {
+import { COUNTRY_OPTIONS } from './countries';
```

Comment:
Do you think it would make sense to fetch this from the server too? I’m personally unsure.

Reply:
Good question. I don't have a strong argument for or against either approach, so would suggest going with the easiest approach.